app/Models/User.php

```diff
@@ -99,4 +99,36 @@ public static function hasNorthstarAccount($type, $id)
 
         return $northstar->getUser($type, $id);
     }
+
+    /**
+     * Create an array with only the useful user information we need for use in views, etc.
+     */
+    public static function setUserInfo($user)
```

Comment:
We can eventually take this out and put to another more accessible class that both the web and api can use in their respective controller :)